manifest.versions.json

```
@@ -15,91 +19,65 @@
     "chisel|8.0|arm|url": "$(chisel|latest|arm|url)",
     "chisel|8.0|arm64|url": "$(chisel|latest|arm64|url)",
-    "chisel|6.0|x64|url": "$(chisel|latest|x64|url)",
-    "chisel|6.0|arm|url": "$(chisel|latest|arm|url)",
-    "chisel|6.0|arm64|url": "$(chisel|latest|arm64|url)",
-
-    "chisel|latest|x64|sha": "5c863d41e9b0829a3af57de444bdb7d1638612e5caa0a289511b5f46ec5728795152381257d30bbff012bcb5c533b6cfbc17b17b93569fd5c0e194ce2c6e0fc5",
-    "chisel|latest|arm|sha": "3e11d2b21eecb1e1ef4414b3aabfdf229f45d1b15f7ebf59fb387db15e055ddbe24dea0c6791219234fecbd27f78e95cb1c499eefcf63742073783ce0d3c2066",
-    "chisel|latest|arm64|sha": "0793124925000cc0103bce1b0f689520866565ba6d0841a3f5dcf4c1c6ffd753d059d9aa068d901f9422b4742fa51c9aa8e848f0751257329a35d99435d5cc1b",
-    "chisel|9.0|x64|sha": "$(chisel|latest|x64|sha)",
-    "chisel|9.0|arm|sha": "$(chisel|latest|arm|sha)",
-    "chisel|9.0|arm64|sha": "$(chisel|latest|arm64|sha)",
-    "chisel|8.0|x64|sha": "$(chisel|latest|x64|sha)",
-    "chisel|8.0|arm|sha": "$(chisel|latest|arm|sha)",
-    "chisel|8.0|arm64|sha": "$(chisel|latest|arm64|sha)",
-    "chisel|6.0|arm|sha": "$(chisel|latest|arm|sha)",
-    "chisel|6.0|x64|sha": "$(chisel|latest|x64|sha)",
-    "chisel|6.0|arm64|sha": "$(chisel|latest|arm64|sha)",
-
-    "dotnet|6.0|product-version": "6.0.36",
-    "dotnet|6.0|fixed-tag": "$(dotnet|6.0|product-version)",
-    "dotnet|6.0|minor-tag": "6.0",
-    "dotnet|6.0|base-url|main": "$(base-url|public|maintenance|main)",
-    "dotnet|6.0|base-url|nightly": "$(base-url|public|maintenance|nightly)",
 
-    "dotnet|8.0|product-version": "8.0.11",
+    "chisel|latest|x64|sha384": "655ca8f45e6f052ab60a4acfeca5017c7d992035e0e4a01f78881c0fe45f2103ce2f18a352ddc8f5fb88bfdd1ed8cc6b",
+    "chisel|latest|arm|sha384": "1bbe68cf9df8a7a2a0c2718bfd28cc68fa5bee424909d01e305480b0510f58fa40e3aa917e077b83445c426317e5772a",
+    "chisel|latest|arm64|sha384": "95bfcff1c4cc27a4649bd8945fe493c9409a3d054100b15dd5d80edc59c0c76c3c57ce153087a4a6411b7b02f9053cab",
+    "chisel|9.0|x64|sha384": "$(chisel|latest|x64|sha384)",
+    "chisel|9.0|arm|sha384": "$(chisel|latest|arm|sha384)",
+    "chisel|9.0|arm64|sha384": "$(chisel|latest|arm64|sha384)",
+    "chisel|8.0|x64|sha384": "$(chisel|latest|x64|sha384)",
+    "chisel|8.0|arm|sha384": "$(chisel|latest|arm|sha384)",
+    "chisel|8.0|arm64|sha384": "$(chisel|latest|arm64|sha384)",
+
+    "dotnet|8.0|product-version": "8.0.12",
     "dotnet|8.0|fixed-tag": "$(dotnet|8.0|product-version)",
     "dotnet|8.0|minor-tag": "8.0",
     "dotnet|8.0|base-url|main": "$(base-url|public|maintenance|main)",
     "dotnet|8.0|base-url|nightly": "$(base-url|public|maintenance|nightly)",
 
-    "dotnet|9.0|product-version": "9.0.0",
+    "dotnet|9.0|product-version": "9.0.1",
     "dotnet|9.0|fixed-tag": "$(dotnet|9.0|product-version)",
     "dotnet|9.0|minor-tag": "9.0",
     "dotnet|9.0|base-url|main": "$(base-url|public|preview|main)",
```

```suggestion
    "dotnet|9.0|base-url|main": "$(base-url|maintenance|preview|main)",
```
Same comment as https://github.com/dotnet/dotnet-docker/pull/6136/files#r1909815408